frontend/src/pages/index.vue

```diff
@@ -0,0 +1,34 @@
+<!--
+  This file is part of the QuestionPy SDK. (https://questionpy.org)
+  The QuestionPy SDK is free software released under terms of the MIT license. See LICENSE.md.
+  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+-->
+
+<template>
+    <PackageCard class="mb-4" />
+    <ButtonGroup class="mb-4">
+        <IconButton :icon-component="IMdiImport" variant="link" @click="importQuestion">Import question</IconButton>
+        <IconButton :icon-component="IMdiAdd" @click="createQuestion" variant="primary">New question</IconButton>
+    </ButtonGroup>
+    <QuestionList />
+</template>
+
+<script lang="ts" setup>
+import IMdiAdd from '~icons/mdi/add'
+import IMdiImport from '~icons/mdi/import'
+
+import useCreateQuestion from '@/composables/question/useCreateQuestion'
+
+const createQuestion = useCreateQuestion()
+
+function importQuestion() {
+    // TODO: import question
+}
+</script>
+
+<route lang="json">
+{
+    "name": "index",
+    "meta": { "title": "Package preview" }
```

Comment:
Um einheitlich mit den anderen `title`s zu bleiben:
```suggestion
    "meta": { "title": "Package Preview" }
```

Reply:
cb1dc3893c3391ad3d3bb8ffe9b2f191500f669b